stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -37,7 +38,7 @@
     long iteration = 0;
 
     public AbstractWorkerTask() {
-        this.selector = createOperationSelector();
+        this.selector = createOperationSelectorBuilder().build();
```

Comment:
I thought this whole approach should be the lazy bum way, so the `AbstractWorkerTask` should do all the things which are needed for a standard test. This will use an `OperationSelector`, so you should guide the user to provide a builder for it. The `doRun()` receives the selected operation, so the `select()` is also done by the `AbstractWorkerTask`, and I think that is very convenient if you want to write a standard test. If you want to do something funky there is still `@Run` where you can setup everything manually.